Gemfile.lock

```diff
@@ -171,28 +171,9 @@ PATH
     samson_zendesk (0.0.0)
       zendesk_api
 
-GEM
```

Comment:
try with `bundle update --bundler` and see if that goes away ... because when I bundle it, it comes right back

Reply:
That worked perfectly. Thanks, @grosser .